chaosblade-exec-common/src/tests/java/com/alibaba/chaosblade/exec/common/util/StringUtilsTest.java

```diff
@@ -0,0 +1,226 @@
+package com.alibaba.chaosblade.exec.common.util;
+
+import org.junit.Assert;
+import org.junit.Test;
+
+public class StringUtilsTest {
+
+    @org.junit.Test
+    public void testHasLength() {
+        Assert.assertTrue(StringUtils.hasLength("foo"));
+
+        Assert.assertFalse(StringUtils.hasLength(""));
+    }
+
+    @org.junit.Test
+    public void testHasText() {
+        Assert.assertFalse(StringUtils.hasText(" "));
+        Assert.assertFalse(StringUtils.hasText(null));
+
+        Assert.assertTrue(StringUtils.hasText("foo"));
+    }
+
+    @org.junit.Test
+    public void testContainsWhitespace() {
+        Assert.assertFalse(StringUtils.containsWhitespace(null));
+        Assert.assertFalse(StringUtils.containsWhitespace("foo"));
+
+        Assert.assertTrue(StringUtils.containsWhitespace(" "));
+    }
+
+    @Test
+    public void testTrimWhitespace() {
+        Assert.assertNull(StringUtils.trimWhitespace(null));
+
+        Assert.assertEquals("foo", StringUtils.trimWhitespace(" foo "));
+    }
+
+    @Test
+    public void testTrimLeadingWhitespace() {
+        Assert.assertNull(StringUtils.trimLeadingWhitespace(null));
+
+        Assert.assertEquals("foo ", StringUtils.trimLeadingWhitespace(" foo "));
+    }
+
+    @Test
+    public void testTrimLeadingCharacter() {
+        Assert.assertNull(StringUtils.trimLeadingCharacter(null, 'a'));
+
+        Assert.assertEquals("oo",
+                StringUtils.trimLeadingCharacter("foo", 'f'));
+    }
+
+    @Test
+    public void testTrimTrailingWhitespace() {
+        Assert.assertNull(StringUtils.trimTrailingWhitespace(null));
+
+        Assert.assertEquals("foo",
+                StringUtils.trimTrailingWhitespace("foo "));
```

Comment:
The input for the test of trimLeadingWhitespace() was " foo " so it would be good to be consistent ie. provide " foo " to both or "foo " for one and " foo" for the other.

Reply:
Done